raiden/api/rest.py

```diff
@@ -549,6 +549,7 @@ def register_token(
             AlreadyRegisteredTokenAddress,
             InvalidToken,
             AddressWithoutCode,
+            ValueError,
```

Comment:
as mentioned before, let's avoid raising and catching ValueErrors if we can help it